tests/test-pipenv.bsh

```diff
@@ -66,4 +66,40 @@ begin_test "30_get-pipenv"
   # test
   [ "$RESULT" = 'pipenv, version 2020.8.13' ]
 )
-end_test
\ No newline at end of file
+end_test
+
+[ "${BASH_VERSINFO[0]}${BASH_VERSINFO[1]}" -le 42 ] && skip_next_test
+begin_test "Get pipenv non-DRY functions"
+(
+  setup_test
+  declare -A function_prototypes
+
+  functions=(download_to_stdout_wget
+             download_to_stdout_curl
+             download_to_stdout_python
+             download_to_file_python
+             download_to_file
+             #  download_to_stdout_ruby
+             #  download_to_stdout_perl
+            )
+
+  source "${VSI_COMMON_DIR}/docker/recipes/30_get-pipenv"
+
+  # Store copies of functions loaded from VSI_COMMON_DIR
+  for function_name in "${functions[@]}" download_to_stdout_ruby download_to_stdout_perl; do
```

Comment:
The array `functions` is used below to check that the `function_prototypes` are equal to `30-get-pipen` verions. Since these two function are dummied out, I removed them from the `functions` array and instead verify they are not equal, just as a sanity check.